installer/lib/phx_new/generator.ex

```diff
@@ -83,6 +83,28 @@ defmodule Phx.New.Generator do
     end
   end
 
+  def prod_only_config_inject(path, file, to_inject) do
+    file = Path.join(path, file)
+
+    contents =
+      case File.read(file) do
+        {:ok, bin} -> bin
+        {:error, _} -> """
+          import Config
+
+          if config_env() == :prod do
```

Comment:
You need to close the block, otherwise the file will be invalid. :)

```suggestion
          if config_env() == :prod do
          end
```